src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -2314,10 +2319,7 @@ impl WithOptionName for AlterSourceAddSubsourceOptionName {
     /// this value could contain sensitive user data. If you're uncertain, err
     /// on the conservative side and return `true`.
     fn redact_value(&self) -> bool {
-        match self {
-            AlterSourceAddSubsourceOptionName::Details
-            | AlterSourceAddSubsourceOptionName::TextColumns => false,
-        }
+        false
```

Comment:
Can you leave this as an explicit match? There's no guarantee that future options won't need redaction, and the explicit match forces all future additions to consider the question.